### ISSUE TITLE
ATAK developer build crashes frequently, possibly due to JNI Error

### ISSUE THREAD

**Issue by vorobotics** [deptofdefense/AndroidTacticalAssaultKit-CIV#276]
### Description

I installed the developer build (`atak.apk` in the `atak-civ-sdk-4.5.1.13.zip`) in the GitHub releases on my Galaxy Tab S8+ running Android 13. After opening it for about 10-15 seconds, ATAK crashes with no on-screen error message. 

For reference, I have previously installed ATAK on this same device from the Play Store, that version does not crash. 

### Logcat:
I ran `adb logcat atakmap.app.civ:F *:S` to see fatal logs and it reveals this message, followed by a huge native stack dump. 
```
02-05 11:43:18.527 24286 24498 F atakmap.app.civ: java_vm_ext.cc:594] JNI DETECTED ERROR IN APPLICATION: input is not valid Modified UTF-8: illegal continuation byte 0xf2
02-05 11:43:18.527 24286 24498 F atakmap.app.civ: java_vm_ext.cc:594]     string: '��Yt'
02-05 11:43:18.527 24286 24498 F atakmap.app.civ: java_vm_ext.cc:594]     input: '0x1c 0xcd <0xf2> 0x59 0x74'
02-05 11:43:18.527 24286 24498 F atakmap.app.civ: java_vm_ext.cc:594]     in call to NewStringUTF
02-05 11:43:18.527 24286 24498 F atakmap.app.civ: java_vm_ext.cc:594]     from void com.atakmap.map.opengl.GLLabelManager.setTextFormat(long, int, java.lang.String, float, boolean, boolean, boolean, boolean)
```

**Comment by solita-michalguspiel** [comment]
Any discoveries in this topic? I have notice major performance issues when running developer build on a real device. Is it supposed to work only on emulator?

**Comment by solita-michalguspiel** [comment]
My plugin completely crashes after attempting to dipsatch CotEvent using internalDispatcher. This issue only happens if running on real device. On emulator it works smoothly.

`2023-03-08 12:32:50.548  5980-6064  atakmap.app.civ         com.atakmap.app.civ                  A  java_vm_ext.cc:594] JNI DETECTED ERROR IN APPLICATION: input is not valid Modified UTF-8: illegal start byte 0x80
                                                                                                    java_vm_ext.cc:594]     string: '??ray'
                                                                                                    java_vm_ext.cc:594]     input: '<0x80> 0xad 0x72 0x61 0x79'
                                                                                                    java_vm_ext.cc:594]     in call to NewStringUTF
                                                                                                    java_vm_ext.cc:594]     from void com.atakmap.map.opengl.GLLabelManager.setTextFormat(long, int, java.lang.String, float, boolean, boolean, boolean, boolean)`

**Comment by matt-msi** [comment]
We're seeing the same issue in ATAK built from 4.5.1.13 open source, app freezes shortly after launch and ANRs with the same fatal error shown in logcat:

```
03-13 09:21:31.699 F/atakmap.app.ci( 6544): java_vm_ext.cc:577] JNI DETECTED ERROR IN APPLICATION: input is not valid Modified UTF-8: illegal continuation byte 0
03-13 09:21:31.699 F/atakmap.app.ci( 6544): java_vm_ext.cc:577]     string: '“'
03-13 09:21:31.699 F/atakmap.app.ci( 6544): java_vm_ext.cc:577]     input: '0xd2'
03-13 09:21:31.699 F/atakmap.app.ci( 6544): java_vm_ext.cc:577]     in call to NewStringUTF
03-13 09:21:31.699 F/atakmap.app.ci( 6544): java_vm_ext.cc:577]     from void com.atakmap.map.opengl.GLLabelManager.setTextFormat(long, int, java.lang.String, float, boolean, boolean, boolean, boolean)
```

**Comment by matt-msi** [comment]
Possible duplicate of #262.

**Comment by bisgroves** [comment]
This is resolved in ATAK 4.8 and forward.
